src/services/group-service.js

```diff
@@ -0,0 +1,10 @@
+const GROUP_URL = "https://anifansonly-java-server.herokuapp.com"
+
+export const findAllClubs = () =>
+    fetch(`${GROUP_URL}/api/boards`).then(response => response.json())
```

Comment:
This is on me, I changed the endpoint. For consistency and sanity I think we are trying to keep it `groups` on the client side. Can you please rename/refactor to `findAllGroups` and the endpoint will be:
```
/api/groups
```

Additionally for this file can you write accompanying service/reducer for creating a group. Please see the server repo's `ClubController.java` line 20 for the endpoint you will be connecting to.  Thanks!